content/en/blog/2023/sunsetting-opencensus.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Sunsetting OpenCensus
+linkTitle: Sunsetting OpenCensus
+date: 2023-05-01
+author: '[Aaron Abbott](https://github.com/aabmass) (Google)'
+---
+
+In 2019, we announced that OpenTracing and OpenCensus would be merging to form
+the OpenTelemetry project. From the start, we considered OpenTelemetry
+[to be the next major version of both OpenTracing and OpenCensus](https://www.cncf.io/blog/2019/05/21/a-brief-history-of-opentelemetry-so-far/).
+
+We are excited to announce that OpenTelemetry has reached feature parity with
+OpenCensus in C++, .NET, Go, Java, JavaScript, PHP and Python.
+[Stable releases](/docs/instrumentation/#status-and-releases) of both the
+Tracing and Metrics SDKs are available in most of these languages with Go and
+PHP soon to follow. This means that OpenTelemetry can collect and export
+telemetry data with the same level of functionality as OpenCensus. Beyond that,
+OpenTelemetry offers a [richer ecosystem](/ecosystem) of instrumentation
+libraries and exporters, and an
+[active open source community](https://www.cncf.io/blog/2023/01/11/a-look-at-the-2022-velocity-of-cncf-linux-foundation-and-top-30-open-source-projects/).
+
+As a result, we will be archiving all OpenCensus GitHub repositories (with the
+exception of [census-instrumentation/opencensus-python][][^python-timeline]) on
+July 31st, 2023. We are excited to see the
+[long term plan for OpenTelemetry](https://medium.com/opentracing/a-roadmap-to-convergence-b074e5815289)
+coming to fruition and encourage all users of OpenCensus to migrate to
+OpenTelemetry.
+
+## How to Migrate to OpenTelemetry
+
+One of the
+[key goals](https://medium.com/opentracing/merging-opentracing-and-opencensus-f0fe9c7ca6f0)
+of the OpenTelemetry project is to provide backward compatibility with
+OpenCensus and a migration story for existing users.
+
+To help ease the migration path, we have provided backward compatibility bridges
+in [Go][go shim], [Java][java shim], [JavaScript][js shim][^shim-next-release],
+and [Python][python shim][^shim-next-release]. Installing these bridges allows
```

Comment:
When are the bridges expected to be stable? Can we be crystal clear about the current state of these shims and when they will be stable? Azure retirement timeline may depend on it.

Reply:
Agree. Would be good to provide a current status of these bridges and how long they will be maintained (whats the LTS plan).

Reply:
LTS is under https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#long-term-support.

As far as stability, now that the specification is marked as stable, we can get each component to be marked as stable.